dev/github_jira_sync.py

```diff
@@ -77,8 +77,8 @@ def get_jira_prs():
                 result = result + [(jira, pull)]
 
         # Check if there is another page
-        link_header = filter(lambda k: k.startswith("Link"), page.info().headers)[0]
-        if "next" not in link_header:
+        link_headers = filter(lambda k: k.startswith("Link"), page.info().headers)
+        if not link_headers or "next" not in link_headers[0]:
```

Comment:
`not link_headers` is an idiomatic to check if the list is empty. I personally encourage people to use explicit `len(...) > 0` comparison (and arguably other projects do too (?)). But I guess it's my preference just for now. It might be fine.

Reply:
I'll use `len(...) > 0` for the next time~